raiden/tests/integration/test_balance_proof_check.py

```diff
@@ -50,18 +52,18 @@ def test_node_can_settle_if_close_didnt_use_any_balance_proof(
         initiator_app=app0,
         target_app=app1,
         token_address=token_address,
-        amount=1,
-        identifier=1,
+        amount=PaymentAmount(1),
+        identifier=PaymentID(1),
         timeout=network_wait * number_of_nodes,
     )
     # stop app1 - the test uses token_network_contract now
     app1.stop()
     token_network_contract = app1.raiden.proxy_manager.token_network(token_network_address)
     empty_balance_proof = BalanceProof(
         channel_identifier=channel_identifier,
-        token_network_address=to_checksum_address(token_network_contract.address),
+        token_network_address=TokenNetworkAddress(token_network_contract.address),
```

Comment:
Nice find. I had to go check the code to make sure that you did not make a mistake here. How come the test was passing before?

Reply:
Probably because they still got converted into the desired format everywhere where it is actually used (`pack_data`, etc.). I didn't check in detail.